src/App.tsx

```diff
@@ -8,11 +8,47 @@ const tabs: Tab[] = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-    <div>{tabs.length}</div>
-  </div>
-);
+type State = {
+  selectedTab: Tab | null;
+  selectedTabId: string;
+};
+
+class App extends React.Component<{}, State> {
+  state: State = {
+    selectedTab: tabs[0],
+    selectedTabId: tabs[0].id,
+  };
+
+  onTabSelected = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { id } = event.target;
+
+    const currentTab = tabs.find(tab => tab.id === id) || null;
```

Comment:
set the first tab if nothing was found

![image](https://user-images.githubusercontent.com/64437769/151826010-b36194f7-86ea-4138-a604-34b7de90e746.png)